docs/docs/tutorials/fashion-mnist-Yarn.md

```diff
@@ -0,0 +1,179 @@
+In this tutorial, you will learn how to build, submit and execute a PyTorch program with BigDL Orca PySpark Estimator on Yarn. 
+
+In particular, we will show you:
+1. How to prepare for the environment;
+2. How to use BigDL orca estimator to conduct a PyTorch example;
+3. How to run the BigDL program on Yarn.
+
+Let's get started!
+
+# Prepare Env
+We need first to use conda to prepare the Python environment on the local machine where we submit our application. Create a conda environment, install BigDL and all the needed Python libraries in the created conda environment:
+
+``` bash
+conda create -n bigdl python=3.7  # "bigdl" is conda environment name, you can use any name you like.
+conda activate bigdl
+
+pip install bigdl
+# Use conda or pip to install all the needed Python dependencies in the created conda environment.
+```
+
+Check the Hadoop setup and configurations of our cluster. Make sure we correctly set the environment variable HADOOP_CONF_DIR, which is needed to initialize Spark on YARN:
+
+```bash
+export HADOOP_CONF_DIR=the directory of the hadoop and yarn configurations
+```
+
+# Training the Fashion-MNIST Example
```

Comment:
There is no need to describe how to write the code; the assumption is that the users have built their code, and need to run it on YARN cluster in a distributed fashion. You may need to discuss what is need for running the code on YARN (e.g., `init_orca_context`, distributed data storage, etc.)